lib/smart_proxy_openscap/openscap_plugin.rb

```diff
@@ -19,6 +19,7 @@ class Plugin < ::Proxy::Plugin
 
     default_settings :spooldir => '/var/spool/foreman-proxy/openscap',
                      :openscap_send_log_file => 'logs/openscap-send.log',
-                     :contentdir => 'openscap/content'
+                     :contentdir => 'openscap/content',
+                     :reportsdir => 'openscap/reports'
```

Comment:
this PR is already too big, but let's refactor this to underscores in future :-) I know it came from previous version but we shouldn't at least add more unless necessary